guides/create_an_app/3-persistence.md

```diff
@@ -1,3 +1,5 @@
-# 4. Design
+# 2. Persistence
```

Comment:
there is number mismatch between file number and code

Reply:
Yep, but only when diffing 🤔